docs/releases/2.16.md

```diff
@@ -116,7 +116,7 @@ This release upgrades the [django-taggit](https://django-taggit.readthedocs.io/e
 
 ### `wagtail.admin.views.generic.DeleteView` follows Django 4.0 conventions
 
-The internal (undocumented) class-based view `wagtail.admin.views.generic.DeleteView` has been updated to align with [Django 4.0's `DeleteView` implementation](https://docs.djangoproject.com/en/4.0/releases/4.0/#deleteview-changes), which uses `FormMixin` to handle POST requests. Any custom deletion logic in `delete()` handlers should be moved to `form_valid()`.
+The internal (undocumented) class-based view `wagtail.admin.views.generic.DeleteView` has been updated to align with [Django 4.0's `DeleteView` implementation](https://docs.djangoproject.com/en/stable/releases/4.0/#deleteview-changes), which uses `FormMixin` to handle POST requests. Any custom deletion logic in `delete()` handlers should be moved to `form_valid()`.
```

Comment:
Django's release notes tend to not have specific Sphinx targets for their sections, so we need to use full URLs. I'm just updating the link to use `stable` instead of the 4.0 version (but still the correct 4.0 notes)